docs/src/app/components/pages/components/RadioButton/ExampleSimple.jsx

```diff
@@ -30,6 +32,13 @@ const RadioButtonExampleSimple = () => (
         disabled={true}
         style={styles.radioButton}
       />
+      <RadioButton
+        value="ludicrous"
```

Comment:
Yeah, I was thinking it might not be a bad idea. without it there is no way to demonstrate how a disabled checked version would look like. Maybe I should add a disabled checked one? Or you think it's ok if we don't demonstrate that?